src/view/utils.js

```diff
@@ -48,4 +48,11 @@ export function getVm(n) {
 	while (n.vm == null && n.parent)
 		n = n.parent;
 	return n.vm;
+}
+
+export function getVnode(el) {
+	el = el || emptyObj;
+	while (el._node == null && el.parent)
```

Comment:
`el.parent` isn't a thing, did you mean `el.parentNode`?

Reply:
Oh, crap. Good catch! 

That's got me many times because all the other functions for relational nodes in Node do not end with ...Node (firstChild, lastChild, nextSibling, previousSibling). This is why API consistency is so important to me -- especially in a weakly typed or untyped language. It's particularly important in JS where things you reference might be missing and mistyped names aren't caught at the reference point.

Reply:
Resolved by commit 0801af0cd3f9f15bda56deee16a978f033598643.